megamek/src/megamek/common/Player.java

```diff
@@ -507,6 +576,32 @@ public int getFledBV() {
         return bv;
     }
 
+    public int getFledUnitsCount() {
+        //TODO: I'm not sure how squadrons are treated here - see getBV()
+        Enumeration<Entity> fledUnits = game.getRetreatedEntities();
```

Comment:
If you want, this would absolutely warrant adding a method to Game that returns a List instead of that silly enumeration and using that to rewrite this method.